### PR TITLE
fix typos with misspell

### PR DESCRIPTION
**Pull request by aspala**
This PR is part of a campaign to fix a lot of typos on github!
You can see the progress on https://github.com/fixTypos/fix_typos/

https://github.com/client9/misspell

### Summary ###
Fix typos

### Checklist ###

* [x] All authors have signed the CLA at https://cla.js.foundation/jquery/jquery
* [ ] New tests have been added to show the fix or feature works
* [ ] Grunt build and unit tests pass locally with these changes
* [ ] If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com

### CODE REVIEW

File: external/qunit/qunit.js [hunk 1]
```diff
@@ -1172,7 +1172,7 @@ Test.prototype = {
 			filter = filter.slice( 1 );
 		}
 
-		// If the filter matches, we need to honour include
+		// If the filter matches, we need to honor include
```
Comment (dmethvin) [inline external/qunit/qunit.js]: anything in `/external` is an external dependency so you'd need to report it in that project.

File: external/sinon/sinon.js [hunk 2]
```diff
@@ -5738,7 +5738,7 @@ if (typeof sinon === "undefined") {
  * Add-on for sinon.fakeServer that automatically handles a fake timer along with
  * the FakeXMLHttpRequest. The direct inspiration for this add-on is jQuery
  * 1.3.x, which does not use xhr object's onreadystatehandler at all - instead,
- * it polls the object for completion with setInterval. Dispite the direct
+ * it polls the object for completion with setInterval. Despite the direct
```
Comment (dmethvin) [inline external/sinon/sinon.js]: anything in `/external` is an external dependency so you'd need to report it in that project.

File: test/data/jquery-1.9.1.js [hunk 3]
```diff
@@ -4328,7 +4328,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 		assert( function( div ) {
 
 			// Select is set to empty string on purpose
-			// This is to test IE's treatment of not explictly
+			// This is to test IE's treatment of not explicitly
```
Comment (dmethvin) [inline test/data/jquery-1.9.1.js]: This is inside a previous release of jQuery used internally so we wouldn't want to change it.

File: test/data/jquery-1.9.1.js [hunk 4]
```diff
@@ -7036,7 +7036,7 @@ jQuery.extend( {
 				value += "px";
 			}
 
-			// Fixes #8908, it can be done more correctly by specifing setters in cssHooks,
+			// Fixes #8908, it can be done more correctly by specifying setters in cssHooks,
```
Comment (dmethvin) [inline test/data/jquery-1.9.1.js]: This is inside a previous release of jQuery used internally so we wouldn't want to change it.

File: test/unit/core.js [hunk 5]
```diff
@@ -813,7 +813,7 @@ QUnit.test( "end()", function( assert ) {
 
 	var x = jQuery( "#yahoo" );
 	x.parent();
-	assert.equal( "Yahoo", jQuery( "#yahoo" ).text(), "check for non-destructive behaviour" );
+	assert.equal( "Yahoo", jQuery( "#yahoo" ).text(), "check for non-destructive behavior" );
```
Comment (dmethvin) [inline test/unit/core.js]: I'm okay with changing British to US spelling, but not particularly bothered about the original.

File: test/unit/manipulation.js [hunk 6]
```diff
@@ -2279,7 +2279,7 @@ QUnit.test( "Cloned, detached HTML5 elems (#10667,10670)", function( assert ) {
 	// First clone
 	$clone = $section.clone();
 
-	// This branch tests a known behaviour in modern browsers that should never fail.
+	// This branch tests a known behavior in modern browsers that should never fail.
```
Comment (dmethvin) [inline test/unit/manipulation.js]: I'm okay with changing British to US spelling, but not particularly bothered about the original.

**Comment by aspala** [comment]
Done, sorry for the inconvenience :)

**Comment by markelog** [comment]
Are we good here?

**Pull request closed without merging**